contracts/ERC7432/NftRolesRegistryVault.sol

```diff
@@ -22,9 +27,20 @@ contract NftRolesRegistryVault is IERC7432 {
     // owner => tokenAddress => operator => isApproved
     mapping(address => mapping(address => mapping(address => bool))) public tokenApprovals;
 
+    constructor() {
+        for (uint256 i = 0; i < allowedRoles.length; i++) {
+            isRoleAllowed[allowedRoles[i]] = true;
+        }
```

Comment:
**suggestion (non-blocking):**  I think it would be great to have a `setAllowedRoles` function to set both `allowedRoles` and `isRoleAllowed` variables.

Reply:
The contract is just showing how to use the EIP, I would avoid adding more functionality (specially since access control might be needed for the setter function)